sdk/servicebus/service-bus/samples/javascript/advanced/deferral.js

```diff
@@ -75,20 +59,20 @@ async function sendMessages() {
   }
   // wait until all the send tasks are complete
   await Promise.all(promises);
-  await queueClient.close();
+  await sender.close();
   await sbClient.close();
 }
 
 async function receiveMessage() {
-  const sbClient = ServiceBusClient.createFromConnectionString(connectionString);
+  const sbClient = new ServiceBusClient(connectionString);
 
-  // If receiving from a Subscription, use `createSubscriptionClient` instead of `createQueueClient`
-  const queueClient = sbClient.createQueueClient(queueName);
+  // If receiving from a subscription you can use the getReceiver(topic, subscription) overload
```

Comment:
Since we are using `queueName`, lets use similar naming for the overload
```suggestion
  // If receiving from a subscription you can use the getReceiver(topicName, subscriptionName) overload
```